### PR TITLE
Added text editor to facilitate add and edit commands

### LINKED ISSUE #64: Add ability to edit contents of cheatsheets

### PR DESCRIPTION
**Pull request by domaxi**

### CODE REVIEW

File: src/main/java/parser/Parser.java [hunk 1]
```diff
@@ -1,17 +1,6 @@
...
-import command.DeleteCommand;
-import command.ExitCommand;
-import command.ListCommand;
-import command.FindCommand;
-import command.HelpCommand;
-import command.ViewCommand;
-import command.FavouriteCommand;
-import command.SettingsCommand;
-import command.CommandEnum;
+import command.*;
```
Comment (abnermtj) [inline src/main/java/parser/Parser.java:3]: I'm not really sure if they would be nitpicky on the use of '*', I've been trying to avoid it because of that.

Comment (domaxi) [reply src/main/java/parser/Parser.java:3]: Did you just list it one by one?

File: src/main/java/parser/Parser.java [hunk 2]
```diff
@@ -88,6 +80,8 @@ private CommandEnum parseTypeOfCommand(String userInput) throws CommandException
             return CommandEnum.CLEAR;
         case "/delete":
             return CommandEnum.DELETE;
+        case "/edit":
+            return CommandEnum.EDIT;
         case "/exit":
             return CommandEnum.EXIT;
         case "/find":
```
Comment (abnermtj) [inline src/main/java/parser/Parser.java:87]: Code looks clean, LGTM!

File: src/main/java/parser/Parser.java (hunk 1 above)
Comment (abnermtj) [reply src/main/java/parser/Parser.java:3]: i think so but it's really ugly and I hate it

**Pull request merged**